packages/react-sdk/src/utils/indexedDB.ts

```diff
@@ -19,12 +21,12 @@ export const openIndexedDB = async () => {
   });
 };
 
-export const getItemFromIndexedDB = async (storeName: string) => {
+export const getItemFromIndexedDB = async (storeName: string, key?: IDBValidKey) => {
   const db = await openIndexedDB();
   return new Promise<any>((resolve, reject) => {
     const transaction = db.transaction(storeName, 'readonly');
     const store = transaction.objectStore(storeName);
-    const request = store.get(1);
+    const request = key ? store.get(key) : store.get(1);
```

Comment:
What does this fallback logic do?

Reply:
pushed new changes that enforce the key param in the function so we don't need the fallback logic now.